packages/supermassive/src/schema/definition.ts

```diff
@@ -103,18 +107,67 @@ const enum InputValueKeys {
 export type InputValueDefinition = TypeReference | InputValueDefinitionTuple;
 export type InputValueDefinitionRecord = Record<string, InputValueDefinition>;
 
+const DirectiveLocation = {
+  QUERY: 0,
```

Comment:
From my experience - using `0` could be a footgun :) One-indexed is probably safer? But up to you